### PR TITLE
[ADD] add module product_form_pricelist

### PR DESCRIPTION
**Pull request by sebastienbeau**
@bealdav pricelist on template are back again ;)

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Yeah, not sure why they removed it. This can be done for v13 that is the same case.

File: product_form_pricelist/models/res_config_settings.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    group_no_list_price = fields.Boolean(
+        "Hide field 'Sales price'",
+        help=(
+            "Tic this option, if you only when to use fixed price and hide "
```
Comment (hparfr) [inline product_form_pricelist/models/res_config_settings.py]:
```suggestion
            "Use fixed price and hide "
```

File: product_form_pricelist/readme/CONFIGURATION.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+If you only want to use fixed price on product and hide the "Sales price" field
+You can tic the option in the sale config menu
```
Comment (hparfr) [inline product_form_pricelist/readme/CONFIGURATION.rst]:
```suggestion
You can tick the option in the sale config menu
```

File: product_form_pricelist/views/product_template_view.xml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+<odoo>
+
+<record id="product_template_view_form" model="ir.ui.view">
+    <field name="model">product.template</field>
+    <field name="inherit_id" ref="product.product_template_form_view" />
+    <field name="arch" type="xml">
+        <button name="open_pricelist_rules" position="attributes">
+            <attribute name="invisible">1</attribute>
+        </button>
+        <xpath expr="//page[@name='general_information']/group" position="after">
```
Comment (hparfr) [inline product_form_pricelist/views/product_template_view.xml]: I think it should be in sales page

**Review by hparfr — CHANGES_REQUESTED** [review]

**Review by ferranmateu — APPROVED** [review]
Functional review OK!

**Comment by xavier-bouquiaux** [comment]
I tested the fixed_pricelist_item_ids from product_template.py and found no problems
from my tests i can say that it works as item_ids (present in previous version of odoo)

File: product_form_pricelist/models/product_pricelist_item.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        store=True,
+    )
+    product_tmpl_id = fields.Many2one(
+        compute="_compute_applied_on_and_tmpl",
+        readonly=False,
+        store=True,
+    )
+
+    @api.depends("product_id", "product_tmpl_id")
+    def _compute_applied_on_and_tmpl(self):
```
Comment (rousseldenis) [inline product_form_pricelist/models/product_pricelist_item.py:23]: @sebastienbeau How does this behaviour fit standard code as additional values exist in Selection ???

https://github.com/odoo/odoo/blob/14.0/addons/product/models/product_pricelist.py#L385

Comment (rousseldenis) [reply product_form_pricelist/models/product_pricelist_item.py:23]: Ok, tested in interface, understood

**Comment by GSLabIt** [comment]
is this ready to merge?

**Comment by sebastienbeau** [comment]
yes, let's merge it

**Pull request merged**